### PR TITLE
athena: configure limits in examples

### PR DESCRIPTION
**Pull request by tobiaszheller**
This PR adds reasonable default to example athena infrastructure script. 
It also adds `bucket_key_enabled` setting on s3 buckets.

### CODE REVIEW

File: examples/athena/variables.tf [hunk 1]
```diff
@@ -43,3 +43,29 @@ variable "table_name" {
...
+  default     = 20
+}
+
+variable "search_event_limiter_time" {
+  description = "Duration between the addition of tokens to the bucket for rate limit used on top of search event API"
+  default     = "1m"
+}
+
+variable "search_event_limiter_amount" {
+  description = "No of tokens that are added to the bucket during specifiec internal for rate limit used on top of search event API"
```
Comment (logand22) [inline examples/athena/variables.tf:69]:
```suggestion
  description = "Number of tokens added to the bucket during specific interval for rate limit used on top of search event API"
```

File: examples/athena/variables.tf [hunk 2]
```diff
@@ -43,3 +43,29 @@ variable "table_name" {
...
+}
+
+# search_event_limiter variables allows to configured rate limit on top of
+# search events API to prevent increasing costs in case of aggressive use of API.
+# In current version Athena Audit logger is not prepared for polling of API.
+# Burst=20, time=1m and amount=5, means that you can do 20 requests without any
+# throtling, next requests will be thottled, and tokens will be filled to
+# rate limit bucket at amount 5 every 1m.
+variable "search_event_limiter_burst" {
+  description = "No of tokens available for rate limit used on top of search event API"
```
Comment (logand22) [inline examples/athena/variables.tf:59]:
```suggestion
  description = "Number of tokens available for rate limit used on top of search event API"
```

**Review by logand22 — APPROVED** [review]

File: examples/athena/athena.tf
[the remark this answers is not in the document]
Comment (tobiaszheller) [reply examples/athena/athena.tf:1]: added, thanks

**Review by strideynet — APPROVED** [review]

**Comment by tobiaszheller** [comment]
@AntonAM will you be able to take a look at it?

**Review by AntonAM — APPROVED** [review]

**Pull request merged**